ReactWindows/ReactNative.Shared/Views/Control/ReactControl.cs

```diff
@@ -59,5 +61,15 @@ public int[] HandledKeyUpKeys
             get;
             set;
         }
+
+        // TODO: implement runtime change raising event to screen reader #1562
+        /// <inheritdoc />                                                    
+        public AccessibilityTrait[] AccessibilityTraits { get; set; }
+
+        /// <inheritdoc />                                              
+        protected override AutomationPeer OnCreateAutomationPeer()
```

Comment:
I thin we may not need the "trick" with setting AutomationProperty.Name for ReactControl to make it visible as AutomationPeer since this will guarantee that automation peer is always created. So, based on how we merge this we can take a note to remove that code

Reply:
I would not optimize the trick in importantForAccessibility implementation out yet, because traits implementation is not dependent on your importantForAccessibility implementation, and there is a possibility, I hope, that it will be replaced by some for strict traits implementation like having component with explicit behavior for each trait.

---
In reply to: [168861563](https://github.com/gmangov/react-native-windows/pull/1#discussion_r168861563) [](ancestors = 168861563)